### PR TITLE
student-home-page and tests

### LINKED ISSUE #255: (SP 0.5)Implement Student Home Page
The tutor's homepage should match the Figma layout.

Image

Image

### PR DESCRIPTION
**Pull request by OliaHrytsak**
For implementing Student-step I've used existing features and makes small changes.

image

image

### CODE REVIEW

File: tests/unit/pages/student-home/StudentHome.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    expect(pageWrapper).toBeInTheDocument()
+    expect(appNetworkCard).toBeInTheDocument()
+    expect(popularCategories).toBeInTheDocument()
+    expect(faq).toBeInTheDocument()
+    expect(howItWorks).toBeInTheDocument()
+  })
+})
+
+describe('TutorHomePage the modal window test', () => {
```
Comment (VictoriaRodik) [inline tests/unit/pages/student-home/StudentHome.spec.jsx]: Change the title of the test to Student's page

File: src/pages/student-home/StudentHome.jsx [hunk 2]
```diff
@@ -29,14 +33,18 @@ const StudentHome = () => {
...
   return (
-    <Box
-      sx={{ backgroundColor: 'backgroundColor', flex: 1, overflowY: 'auto' }}
-    >
-      <Container data-testid='studentHome' sx={{ pt: 6 }}>
+    <PageWrapper>
+      <Container data-testid='studentHome' sx={styles.container}>
+        <AppNetworkCard userRole={userRole} />
+        <PopularCategories
+          description='tutorHomePage.popularCategories.description'
```
Comment (VictoriaRodik) [inline src/pages/student-home/StudentHome.jsx]: change description if exists

File: tests/unit/pages/student-home/StudentHome.spec.jsx (hunk 1 above)
Comment (OliaHrytsak) [reply tests/unit/pages/student-home/StudentHome.spec.jsx]: Thank you! I've made this changes

File: src/pages/student-home/StudentHome.jsx (hunk 2 above)
Comment (OliaHrytsak) [reply src/pages/student-home/StudentHome.jsx]: Thank you! I've made this changes

**Review by iaremchuk — APPROVED** [review]

**Review by StasRusnak1 — APPROVED** [review]

**Review by Wiret02 — APPROVED** [review]

**Review by iva-stasia — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**